### PR TITLE
Check rancher status

### PR DESCRIPTION
**Pull request by futuretea**
Add a Ready  status (harvester and rancher  are both Ready) to prompt the correct time to log in

image

**Related issue**
rancher/harvester#597

### CODE REVIEW

File: pkg/console/dashboard_panels.go [hunk 1]
```diff
@@ -392,16 +403,39 @@ func getHarvesterStatus() string {
 		return wrapColor(statusUnknown, colorYellow)
 	}
 
-	status, err := harvesterPodStatus()
+	harvesterRunning, harvesterReady, harvesterStatus, err := getPodStatus("harvester-system", "app.kubernetes.io/name=harvester")
```
Comment (gitlawr) [inline pkg/console/dashboard_panels.go:406]: I don't think users can distinguish `running` and `ready` without knowledge of pod status.

Comment (futuretea) [reply pkg/console/dashboard_panels.go:406]: Ready -> green color
Running -> yellow color

**Comment by futuretea** [comment]
Updated

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**